utils/growpart/Makefile

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=growpart
+PKG_SOURCE_DATE:=2022-02-04
+PKG_SOURCE_VERSION:=4cc4950394b3db0243cb0211d6a394d629129a2a
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_SOURCE_DATE).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/canonical/cloud-utils/tar.gz/$(PKG_SOURCE_VERSION)?
+PKG_HASH:=291b045d550d07b7271ce61d5912962b031df53dfa27edce6821b9222b880b3b
+
+PKG_MAINTAINER:=Eero Vuojolahti <eero@oittaa.net>
+PKG_LICENSE:=GPL-3.0
```

Comment:
This one is old SPDX License Identifier. Please check https://spdx.org/licenses/

Reply:
All these comments that are basically "RTFM!!1!" are quite worthless when your documentation is outdated or non-existent. I copied the headers from some other package. Please fix your documentation if it's wrong. That link tells nothing what needs to be put exactly in the file.

Reply:
[CONTRIBUTING.md](https://github.com/openwrt/packages/blob/0bc25077b37d74065fd107dc9f85fbb11ccba56b/CONTRIBUTING.md#makefile-contents-should-contain) has this:

> A PKG_LICENSE tag declaring the main license of the package. (E.g.: PKG_LICENSE:=GPL-2.0-or-later) Please use SPDX identifiers if possible (see list at the bottom).

The SPDX identifiers for GPL 3.0 are `GPL-3.0-only` and `GPL-3.0-or-later`. It is up to you to figure out if the upstream project requires GPL 3.0 only or is compatible with 3.0 or later. (The script files in upstream are a bit of a mess, but [these lines](https://github.com/canonical/cloud-utils/blob/5c24f6a24c5dc8df48b17ff1a635ce047210a160/bin/growpart#L8-L10) are pretty clear.)

> I copied the headers from some other package.

Just because you copied them out of another package doesn't mean they are correct.

> All these comments that are basically "RTFM!!1!" are quite worthless when your documentation is outdated or non-existent. ... Please fix your documentation if it's wrong.

This project is run by (mostly) volunteers. The documentation is (almost certainly all) written by volunteers. If the documentation is wrong then perhaps you can help by fixing it.